### PR TITLE
Getter [any] evaluation fix for primitive arrays

### PR DESCRIPTION
**Pull request by tombujok**

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/query/impl/getters/AbstractMultiValueGetter.java [hunk 1]
```diff
@@ -213,4 +228,8 @@ static void validateModifier(String modifier) {
         parseModifier(modifier);
     }
 
+    public static void main(String[] args) {
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/query/impl/getters/AbstractMultiValueGetter.java]: leftover?

**Comment by jerrinot** [comment]
backport to 3.8 ?

File: hazelcast/src/main/java/com/hazelcast/query/impl/getters/AbstractMultiValueGetter.java (hunk 1 above)
Comment (tombujok) [inline hazelcast/src/main/java/com/hazelcast/query/impl/getters/AbstractMultiValueGetter.java]: Yep, fixed.

**Comment by tombujok** [comment]
Backport https://github.com/hazelcast/hazelcast/pull/9902

**Pull request merged**